nbs/models.ipynb

```diff
@@ -26,7 +26,7 @@
    "source": [
```

Comment:
<p>I like this new format since it's more clear and detailed. </p><p> </p><p>I'll ask Fede how to merge this with what I've been working on since we did some changes to the fit, predict, and forecast functions to be able to compute the prediction intervals for the Historic Average, Naive, RWD, and Seasonal Naive. </p><p> </p><p>Will use this new format for Holt and Holt-Winters, which are currently in progress. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Super cool.

Yes, the docstrings flexibility is not yet matched by `fastcore`'s methods.
Also the main `.py` code was kept without documentation if we relied on them.

Hope that the merge with the new prediction intervals and the Holt-Winters methods is easy.